include/platform/clinician_input.h

```diff
@@ -13,3 +13,13 @@
  *
  */
 status_t get_respiratory_rate(uint32_t * RR);
+
+/*
+ * Updates <selected inspiratory time> variable insp_time 
```

Comment:
I am not sure that the `< ... >` makes sense in the context of this code. The reader would have to know what this means and that it is a reference to the SRS. Is there a better way you can indicate this? 

I think you could just drop the `< ... >` and it would preserve its meaning.